test/integration/smoke/test_vm_lifecycle_with_snapshot_or_volume.py

```diff
@@ -311,6 +306,7 @@ def create_volume_from_snapshot_deploy_vm(self, snapshotid):
                                                 volumeid=volume.id,
                                                 mode="basic",
                                                 )
+        self._cleanup.append(virtual_machine)
```

Comment:
```suggestion
        self.cleanup.append(virtual_machine)
```
?
(as it is an operation on self and not on cls)